packages/picker/src/Picker.ts

```diff
@@ -457,8 +443,8 @@ export class PickerBase extends SizedMixin(Focusable) {
                 <sp-menu
                     id="menu"
                     role="${this.listRole}"
-                    selects="single"
                     @change=${this.handleChange}
+                    .selects=${this.selects}
```

Comment:
It looks like this is intentionally setting a property rather than an attribute. Could you share the motivation?

Reply:
In this case the property is used to overcome the typing. Typescript likes to point out that `this.selects` can be `undefined` which is not an acceptable type for the `selects` attribute, and would prefer `ifDefined` be used here to manage that. However, `Menu.selects` is a reflecting property, so it we know that it will do the work of `ifDefined` internally when supplied `undefined` as a property, so this intends to simplify the usage.

Reply:
Gotcha, thanks!